### ISSUE TITLE
Form wizard within CMS-integrations

### ISSUE THREAD

**Issue by matacino** [barseghyanartur/django-fobi#227]
Hi,

I try tu use django-fobi within wagtail-CMS.
It looks all quite smooth except I cant use form-wizards inside the CMS (means I can't select them in the wagtail-fobi-page - only "simple" forms).
I have had looks inside the code for the integrations of wagtail, feinCMS or Djangocms and there is always just the "fobi.FormEntry" connected with the CMS never the "fobi.FormWizardFormEntry".

1. Is it by design?
2. Is it a bug?
3. Or - Do I miss something?

Thanx in advance for an answer!

**Comment by barseghyanartur** [comment]
It's not yet implemented.

**Comment by matacino** [comment]
Thank you for the answer! In witch way is it not implemented? Does the integration needs enhancement or the Form(-wizard)-Display-mechanism?

**Comment by barseghyanartur** [comment]
@matacino:

Yes, exactly. That part is totally missing in the integration (for all sorts, thus Wagtail, Django CMS, FeinCMS, Django REST Framework and FeinCMS).
